packages/components/src/components/tab/tab.component.ts

```diff
@@ -120,6 +122,15 @@ class Tab extends IconNameMixin(Buttonsimple) {
   @property({ type: String, reflect: true })
   variant: Variant = DEFAULTS.VARIANT;
 
+  /**
+   * Id of the tab (used as a identificator when used in the tablist)
+   * Note: has to be unique!
+   *
+   * @default undefined
+   */
+  @property({ type: String, reflect: true, attribute: 'tabid' })
```

Comment:
```suggestion
  @property({ type: String, reflect: true, attribute: 'tab-id' })
```

We would need to name it in this way so that it is consistent with the native HTML attributes.
This also resolved showing duplicate property value on storybook.

Reply:
changed